docs/user/alerting/rule-types/geo-rule-types.asciidoc

```diff
@@ -10,24 +10,90 @@
 
-- *Entities index*: An index containing a `geo_point` or `geo_shape` field, `date` field, and entity identifier. An entity identifier is a `keyword`, `number`, or `ip` field that identifies the entity. Entity data is expected to be updating so that there are entity movements to alert upon.
-- *Boundaries index*: An index containing `geo_shape` data.
-Boundaries data is expected to be static (not updating). Boundaries are collected once when the rule is created and anytime after when boundary configuration is modified.
+. Set the check interval, which defines how often to evaluate the rule conditions. 
 
-Entity locations are queried to determine if they are contained within any monitored boundaries.
-Entity data should be somewhat "real time", meaning the dates of new documents aren’t older
-than the current time minus the amount of the interval. If data older than
-`now - <current interval>` is ingested, it won't trigger a rule.
+Entity locations are queried to determine whether they are contained within any monitored boundaries.
+Entity data should be somewhat "real time", meaning the dates of new documents aren't older than the current time minus the amount of the interval.
+If data older than `now - <current interval>` is ingested, it won't trigger a rule.
 
 [float]
-=== Actions
+=== Add actions
 
-A rule can be triggered either when a containment condition is met or when an entity is no longer contained.
+You can optionally send notifications when the rule conditions are met.
+In particular, this rule type supports:
+
+- alert summaries
+- actions that run when the containment condition is met
+- actions that run when an entity is no longer contained
+
+For each action, you must choose a connector, which provides connection information for a {kib} service or third party integration.
+For more information about all the supported connectors, go to <<action-types>>.
+
+After you select a connector, you must set the action frequency.
+You can choose to create a summary of alerts on each check interval or on a custom interval.
+Alternatively, you can set the action frequency such that actions run for each alert.
+Choose how often the action runs (at each check interval, only when the alert status changes, or at a custom action interval).
+You must also choose an action group, which indicates whether the action runs when the containment condition is met or when an entity is no longer contained.
+Each connector supports a specific set of actions for each action group.
+For example:
 
 [role="screenshot"]
 image::user/alerting/images/alert-types-tracking-containment-action-options.png[Action frequency options for an action,width=75%]
 // NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+You can further refine the conditions under which actions run by specifying that actions only run they match a KQL query or when an alert occurs within a specific time frame.
```

Comment:
"specifying that actions only run they match a KQL query" -> "specifying that actions only run when they match a KQL query", adding "when" between "only run they match"

Reply:
Good catch!  Fixed in 0089bf11be8315399848ffedbe7b757f28ac1a95